src/components/templates/AvatarGrid/AvatarGrid.tsx

```diff
@@ -0,0 +1,278 @@
+import UserProfilePicture from "components/molecules/UserProfilePicture";
+import firebase from "firebase/app";
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
+import React, { useCallback, useEffect, useState } from "react";
+import { WithId } from "utils/id";
+import { User } from "types/User";
+import { useParams } from "react-router-dom";
+import "./AvatarGrid.scss";
+import UserProfileModal from "components/organisms/UserProfileModal";
+import { AvatarGridRoom } from "types/AvatarGrid";
+import { RoomModal } from "./RoomModal";
+import ChatDrawer from "components/organisms/ChatDrawer";
+import PARTY_BACKGROUND from "./party-bg.jpg";
+
+type Props = {
+  venueName: string;
+};
+
+const DEFAULT_COLUMNS = 40;
+const DEFAULT_ROWS = 25;
+
+const WIDTH = 2451;
```

Comment:
Just wondering - how will this make the grid look on mobile? What about on an iPad, or an old laptop? Is fixed width and height the right approach here? Can we use reponsive breakpoints or flex with max-width so the grid cells shrink but the grid scrolls vertically, for example?